release/rllib_tests/app_config.yaml

```diff
@@ -1,4 +1,4 @@
-base_image: "anyscale/ray-ml:pinned-nightly-py37-gpu"
+base_image: "anyscale/ray-ml:pinned-nightly-py38-gpu"
```

Comment:
Is this required? Otherwise we should stick with a common base image across tests

Reply:
+1
I'll revert

Reply:
```suggestion
base_image: "anyscale/ray-ml:pinned-nightly-py37-gpu"
```